models/choral.js

```diff
@@ -68,13 +68,20 @@ choralSchema.statics.createNew = function (attrs, cb) {
   });
 };
 
-choralSchema.statics.findAllForUser = function (user, cb) {
+choralSchema.statics.findChoralsForUser = function (user, cb) {
   this.find({ userId: user._id }, (err, chorals) => {
```

Comment:
If we are going to have devices as well as chorals we should only return chorals on this page. We also will still probably need a find all for user when adding children.